dbreps2/src/lib.rs

```diff
@@ -192,6 +192,9 @@ pub trait Report<T: Send + Sync> {
                 return Ok(true);
             }
         }
+        if contains_database_report_template(old_text) {
+            return Ok(false);
+        }
```

Comment:
Please document why. For example something along the lines of
```
// Pages with {{database report}} are maintained by SDZeroBot and contains
// the SQL code directly. Skip reporting if it has been migrated to use that template.
```

Reply:
Done